### DISCUSSION TITLE
TuringCourses/major_mandatory/computer_vision/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#179] [General]
# TuringCourses/major_mandatory/computer_vision/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/major_mandatory/computer_vision/

**Comment by MTDickens** [comment]
有兴趣可以看看 Deep Learning for Computer Vision by Justin Johnson。https://web.eecs.umich.edu/~justincj/teaching/eecs498/FA2020/
https://www.bilibili.com/video/BV1zg411a7Wi/

**Comment by cxzhou35** [comment]
想要进一步了解和学习3DV的同学建议看tuebingen的这门课：[Computer Vision](https://uni-tuebingen.de/fakultaeten/mathematisch-naturwissenschaftliche-fakultaet/fachbereiche/informatik/lehrstuehle/autonomous-vision/lectures/computer-vision/) ：）

**Comment by ken0yuan** [comment]
表面重建在网上有详细的教程，是最简单的期末大程，先展示有一定的加分，建议早做，训练一次模型用服务器需要10小时，建议减少迭代次数训练3小时看看情况，期末中文比较简单，参考98回忆卷，重合度较高，如果好好上课理解大纲期末不需要复习什么的。
